### PR TITLE
Add basic expanded sflow tests

### PR DESCRIPTION
**Pull request by simPod**
I have been debugging expanded sflow parsing and for that I wrote these 2 tests.

(I was not sure it was being parsed properly. The error in the end was in reading prometheus metrics (wrong label filter) so everything in goflow works as expected)

I though might share rather than removing it, WDYT?

### CODE REVIEW

File: utils/sflow_test.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+func TestDecodeFlowExpandedSFlow(t *testing.T) {
+	msg := BaseMessage{
+		Src:     []byte{},
+		Port:    1,
+		Payload: getExpandedSFlowDecode(),
+	}
+
+	s := &StateSFlow{
+		Transport: &DefaultLogTransport{},
+		Logger:    log.StandardLogger(),
```
Comment (lspgn) [inline utils/sflow_test.go]: Nitpicking: I'd rather not have the import of a library for logging in tests.

**Comment by lspgn** [comment]
LGTM. Thank you for writing the test :) .

File: utils/sflow_test.go (hunk 1 above)
Comment (simPod) [reply utils/sflow_test.go]: I'm not very familiar with go but seems like I could just remove the line and it works

**Comment by lspgn** [comment]
Sorry I've been taking too long at verifying this PR.

Are you sure you want to share a real sFlow sample? I'm noticing a few ip addresses in there that don't seem to be obfuscated.

**Comment by simPod** [comment]
I merged two flow samples into one so it should not represent real communication. Do you think it is not safe still?

**Comment by lspgn** [comment]
I'd be careful in case of an audit: non-obfuscated PII (even if long ago) since it's the full sFlow packet.
Would prefer not have to clear the whole git history if that were to cause an issue.

**Pull request merged**